### PR TITLE
Feature developer

### PR DESCRIPTION
**Pull request by Fig77**
Pull request for code review.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi Facundo!  
#### Your project looks :+1:  amazing readme file, so descriptive!

#### But project need some Required Changes

#### Page structure

##### Navigation bar

* [x] It contains Newsweek logo
* [x] It contains current date
* [ ] It contains Sign in and Subscribe links. Subscribe should be link too.
* [x] It contains main navigation links
* [x] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))
*** Header should be full screen, but in your version, I see spaces in left and right sides ***
*** Original***

*** Your version ***


##### Main content

* [x] It contains all necessary sections 
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/)). Please check responsiveness:

*** Original***


*** Your version ***


*** Original***


*** Your version ***


*** Original***


*** Your version ***


*** Original***


*** Your version ***


##### Footer

* [x] It contains Newsweek logo
* [x] It contains social media icons
* [x] It contains main navigation links
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

#### Implementation details

* [ ] It references [Bootstrap CSS file](https://getbootstrap.com/docs/4.3/getting-started/introduction/#css) 
* [x] It adds [viewport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag) ([screenshot](/microverse/guides/tse/code_review/code_review_guidelines/raw/master/images/html_css/using_bootstrap/bootstrap_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap))
* [ ] It uses Bootstrap Css classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))


Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

File: index.html [hunk 1]
```diff
@@ -0,0 +1,950 @@
...
+            </section>
+          </div>
+        </div>
+      </div>
+
+      <!-- main-aside-right -->
+
+      <aside class="col-lg-4 col-sm-12 order-2 order-lg-2 order-md-2 order-xl-2 order-sm-2 aside aside-right">
+        <h6><a href="#" class="text-muted font-weight-bold">OPINION</a></h6>
+        <div class="container-fluid pl-0">
```
Comment (suretrust) [inline index.html]: **Medium screen**

- [ ] The opinion section needs some margin-top. Refer to the images below:

**Yours**
opine
**Original**
opine2

File: index.html [hunk 2]
```diff
@@ -0,0 +1,950 @@
...
+    </div>
+  </main>
+
+  <!-- Section In Magazine -->
+
+  <section class="w-100 mr-auto ml-auto">
+    <div class="d-flex justify-content-center align-content-end mr-auto ml-auto mt-4">
+      <i class="n-icon mt-3"></i>
+      <h6 class="text-dark font-weight-bold align-self-end">In the magazine</h6>
+    </div>
```
Comment (suretrust) [inline index.html]: **Medium Screen**

- [ ] The `In the magazine` section needs some margin-top. Refer to the image below:

mag

**Pull request merged**